db-dump-entrypoint.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env sh
+# Alternative entrypoint for the inmanta orchestrator container.
+# Instead of starting the orchestrator, it starts a cron daemon, to allow
+# pg_dump to periodically run and export a database dump.
+
+set -x
+set -e
+
+# Install cron, logrotate and tini
+apt-get update
+apt-get install -y cron logrotate  tini
+
+# Configure db dump script, save container env vars in script to simplify
+# container configuration
+if [ ! -f /db_dump ]; then
+    cat > /db_dump <<EOF
+#!/usr/bin/env sh
+$(export)
+pg_dump -f /var/lib/postgresql/data/db_dump.sql
+EOF
+	chmod 755 /db_dump
+fi
+
+# Use logrotate to keep multiple backups, compress them, and don't
+# leak too much old data
+if [ ! -f /etc/logrotate.d/postgres ]; then
+    cat > /etc/logrotate.d/postgres <<EOF
+/var/lib/postgresql/data/*.sql {
+	daily
+	compress
+	rotate 10
+	missingok
+	create 0644 postgres postgres
+    postrotate
+    su -c /db_dump - postgres
+    endscript
+}
+EOF
+fi
```

Comment:
I use logrotate here to keep multiple backups on the host, while avoiding occupying too much disk space.  I also rely on logrotate to trigger the backups (as logrotate itself is triggered by cron)